lib/resources/remote_config.dart

```diff
@@ -14,6 +16,7 @@ class RemoteConfigService {
       : _remoteConfig?.getBool(_feedbackEnabled) ?? defaults[_feedbackEnabled];
 
   static Future<dynamic> initialize() async {
+    if (!Platform.isAndroid && !Platform.isIOS) return;
```

Comment:
Just:
```suggestion
    if (kIsWeb) return; // Remote config is not yet supported on web.
```

And remove the comment below.